<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>后台管理界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="menu_btn">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>菜单管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="user_btn">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="order_btn">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>订单信息</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="back">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>返回登录</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="DataTableWidget" name="infoTable" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="save">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>保存数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fresh">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="help"/>
   </widget>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="update1">
   <property name="text">
    <string>update</string>
   </property>
  </action>
  <action name="add">
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="update2">
   <property name="text">
    <string>update</string>
   </property>
  </action>
  <action name="delete_2">
   <property name="text">
    <string>delete</string>
   </property>
  </action>
  <action name="delete_3">
   <property name="text">
    <string>delete</string>
   </property>
  </action>
  <action name="add_2">
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="help">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actionAllEmpty">
   <property name="text">
    <string>AllEmpty</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DataTableWidget</class>
   <extends>QWidget</extends>
   <header>qtpandas.views.DataTableView</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
